### PR TITLE
chips: Add `qemu.sh` to the chip files.

### PR DESCRIPTION
**Pull request by beezow**
Enables Humility to start any app using that chip within a VM.

### CODE REVIEW

File: build/xtask/src/dist.rs [hunk 1]
```diff
@@ -606,6 +606,13 @@ fn build_archive(cfg: &PackageConfig, image_name: &str) -> Result<()> {
     archive
         .copy(chip_dir.join("openocd.gdb"), debug_dir.join("openocd.gdb"))?;
 
+    // Copy `qemu.sh` into the archive if it exists;
```
Comment (furquan-rivos) [inline build/xtask/src/dist.rs:609]: Is this for humility?

File: chips/fe310-g002/qemu.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+qemu-system-riscv32 -M sifive_u,msel=1 -nographic -serial mon:stdio -device loader,addr=0x20010000,cpu-num=0,file=final.bin -semihosting -semihosting-config enable=on,userspace=on
```
Comment (furquan-rivos) [inline chips/fe310-g002/qemu.sh:1]: For my learning: Does a chip (e.g. fe310-g002) always map to a single machine model(e.g. sifive_u) in qemu?

Comment (FawazTirmizi) [reply chips/fe310-g002/qemu.sh:1]: Probably. The chip is primarily used to determine the availability, base address, and size of MMIO peripherals, so I doubt we'd be seeing many situations where multiple QEMU machines would work out of the box for the same chip.

File: build/xtask/src/dist.rs (hunk 1 above)
Comment (FawazTirmizi) [reply build/xtask/src/dist.rs:609]: Yeah, Humility will need to be able to pull this script out of the archive.

**Review by FawazTirmizi — APPROVED** [review]
LGTM

File: chips/fe310-g002/qemu.sh (hunk 2 above)
Comment (furquan-rivos) [reply chips/fe310-g002/qemu.sh:1]: In that case chip<->machine model can be considered as 1:1. I just wanted to ensure that the terminologies used in hubris and qemu match i.e. chip in hubris is the same as machine model in qemu.

**Review by furquan-rivos — APPROVED** [review]

File: build/xtask/src/dist.rs (hunk 1 above)
Comment (beezow) [reply build/xtask/src/dist.rs:609]: Yea, the archive should include everything humility needs to run.

File: chips/fe310-g002/qemu.sh (hunk 2 above)
Comment (beezow) [reply chips/fe310-g002/qemu.sh:1]: I think that every chip should only have 1 machine model that represents it, but multiple chips may share a machine model.  However, there is probably an exception I don't know about.

**Pull request merged**